<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomaticSettings</class>
 <widget class="QDialog" name="AutomaticSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Media Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="title_label">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Media Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="close_button">
     <property name="text">
      <string>Finished</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="reset_button">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="main">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2"/>
     </widget>
     <widget class="QWidget" name="roi">
      <attribute name="title">
       <string>RoI</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="10" column="1">
        <widget class="QCheckBox" name="ctu_based">
         <property name="text">
          <string>CTU based drawing</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="show_grid">
         <property name="text">
          <string>Show grid</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="VideoWidget" name="roi_surface" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="9">
        <widget class="QGroupBox" name="groupBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>RoI mode</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="roi_off_button">
            <property name="text">
             <string>Off</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="roi_manual_button">
            <property name="text">
             <string>Manual</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="roi_yolo_button">
            <property name="text">
             <string>Object Detection</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="roi_lable">
         <property name="text">
          <string>RoI QP</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="roi_qp">
         <property name="suffix">
          <string/>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>22</number>
         </property>
         <property name="maximum">
          <number>47</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="background_label">
         <property name="text">
          <string>Global QP</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="background_qp">
         <property name="minimum">
          <number>22</number>
         </property>
         <property name="maximum">
          <number>47</number>
         </property>
         <property name="value">
          <number>47</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="brush_size_label">
         <property name="text">
          <string>Brush Size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="brush_size">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>3</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="objectSelection">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="1" rowspan="3" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>ui/gui/videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
